.github/workflows/verify.yaml

```diff
@@ -197,3 +187,242 @@ jobs:
           kubectl get pods
           kubectl logs deployment/provider-dataspace-connector --tail=200
           kubectl logs deployment/consumer-dataspace-connector --tail=200
+
+  Azure-Storage-Integration-Tests:
```

Comment:
is this test needed?

Reply:
Not for our implementation. It comes from upstream. I could delete it, but not exactly sure how much we can deviate from the main repo.